openwisp_monitoring/monitoring/tests/test_admin.py

```diff
@@ -0,0 +1,59 @@
+from django.test import TestCase
+from django.urls import reverse
+from swapper import load_model
+
+from . import TestMonitoringMixin
+
+Notification = load_model('openwisp_notifications', 'Notification')
+
+
+class TestAdmin(TestMonitoringMixin, TestCase):
+    def test_action_disable_notifications(self):
+        m = self._create_general_metric(name='test')
+        self._create_threshold(metric=m, operator='>', value=90, seconds=0)
+        path = reverse('admin:monitoring_metric_changelist')
+        self.client.force_login(self._get_admin())
+        post_data = {
+            '_selected_action': [m.pk],
+            'action': 'disable_notifications',
+            'csrfmiddlewaretoken': 'test',
```

Comment:
csrf validation is disabled when testing in Django, can you confirm it and remove this parameter?

Reply:
Confirmed, thanks for sharing didn't knew that earlier :smiley: